src/logic/operate.js

```diff
@@ -0,0 +1,17 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const num1 = new Big(numberOne);
+  const num2 = new Big(numberTwo);
+
+  switch (operation) {
+    case '+': return num1.plus(num2).toFixed();
+    case '-': return num1.minus(num2).toFixed();
+    case '*': return num1.times(num2).toFixed();
+    case '/': return num1.div(num2).toFixed();
+    case '%': return 100 * num1.div(num2).toFixed();
+    default: return null;
```

Comment:
To evaluate the percentage of a number, you divide the number by 100. That is 
```
case '%': return num1.div(100).toString();
```
Also convert your values to string, using .toString() method.

Reply:
Thank you for the review Henry. I have fixed evaluation of the percentage. I am using .toFixed() method to convert the values to string. I thought it would be better to use it, but I can also use .toString() method. Thanks once more.